source/exe/main_common.cc

```diff
@@ -93,6 +93,18 @@ bool MainCommonBase::run() {
   NOT_REACHED_GCOVR_EXCL_LINE;
 }
 
+void MainCommonBase::adminRequest(absl::string_view path_and_query, absl::string_view method,
```

Comment:
This looks unused.  Was this intentional?  If we're adding a helper for downstream use we should comment, and please test either way.